core/rawdb/accessors_chain.go

```diff
@@ -360,6 +360,20 @@ func ReadHeader(db ethdb.Reader, hash common.Hash, number uint64) *types.Header
 	return header
 }
 
+// ReadHeaderAndRaw retrieves the block header corresponding to the hash.
+func ReadHeaderAndRaw(db ethdb.Reader, hash common.Hash, number uint64) (*types.Header, rlp.RawValue) {
```

Comment:
do we really need this?
replace by ReadHeader, and check if the return value is nil, where ReadHeaderAndRaw used